content/livesync/models.textile

```diff
@@ -263,7 +322,7 @@ model.on('ready', () => { /* model resumed */ });
 model.on('disposed', () => { /* model disposed */ });
 ```
 
-h3(#pause-resume). Pausing and resuming a model
+h3(#pausing-resuming). Pausing and resuming a model
 
 You can pause a model to prevent the model from consuming realtime updates, while reserving the ability to resume it at some point in the future. Pausing can be useful in instances such as when the UI rendering the model data is temporarily out-of-view.
 
```

Comment:
"The following example un-pauses the model. Event processing will resume and changes will be made available to subscribers:" -->

"The following example resumes the model. Event processing will resume and changes will be made available to subscribers:"